package.json

```diff
@@ -6,22 +6,25 @@
   "license": "gimmeAllzDaRoyaltiez",
   "jest": {
     "setupTestFrameworkScriptFile": "jest-extended",
-    "modulePathIgnorePatterns": ["helpers"]
+    "modulePathIgnorePatterns": [
+      "helpers"
+    ]
   },
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "nodemon server/index.js & npm run react-dev | npm run openPage",
+    "start": "nodemon server/index.js | npm run openPage",
```

Comment:
if this is the one we should launch, take off run openpage as other people don't want a browser window.